linear_algebra/code/CMakeLists.txt

```diff
@@ -66,6 +75,17 @@ target_compile_features(wg21_linear_algebra
         cxx_std_17
 )
 
+target_compile_definitions(wg21_linear_algebra
+    INTERFACE
+        LA_USE_MDSPAN
```

Comment:
This automatically defines the "LA_USE_MDSPAN" definition for the project.  The below line does the same, but only if the compiler is not Visual Studio.  I wasn't sure which you would prefer given the header manually undefines LA_USE_MDSPAN if the compiler is Visual Studio.  

Anything defined here will automatically propagate to the generated install targets, so users of the Linear Algebra library would by depending on the library, automatically get this define included for their project.  Let me know if this is not what you want and I can remove or add a CMake option for the user to control this.

Reply:
Let's have it define LA_USE_MDSPAN, and pull down mdspan, for all targets, including VS.  Meanwhile, I will continue to manually disable it by undefining in the linear_algebra.h header for now.  Once I figure out what's going on with VS, I can just remove the undefine.